### PR TITLE
Story/vspc 149

### PR DESCRIPTION
**Pull request by AvirupBiswas**
# Guidelines for Pull Requests

If you haven't yet read our code review guidelines, please do so, You can find them [here](https://diging.atlassian.net/wiki/spaces/DIGING/pages/2256076801/Code+Review+Guidelines).

Please confirm the following by adding an x for each item (turn `[ ]` into `[x]`).

- [x] I have removed all code style changes that are not necessary (e.g. changing blanks across the whole file that don’t need to be changed, adding empty lines in parts other than your own code)
- [x] I am not making any changes to files that don’t have any effect (e.g. imports added that don’t need to be added)
- [x] I do not have any sysout statements in my code or commented out code that isn’t needed anymore
- [x] I am not reformatting any files in the wrong format or without cause. 
- [x] I am not changing file encoding or line endings to something else than UTF-8, LF
- [x] My pull request does not show an insane amount of files being changed although my ticket only requires a few files being changed
- [x] I have added Javadoc/documentation where appropriate
- [x] I have added test cases where appropriate
- [x] I have explained any part of my code/implementation decisions that is not be self-explanatory

## Please provide a brief description of your ticket 
(you can copy the ticket if it hasn't changed) 
Just to check whether the code is flowing in this branch from local
...Put description here...
  
## Anything else the reviewer needs to know?

... describe here ...

### CODE REVIEW

**Comment by jdamerow** [comment]
Resolve conflicts please

File: vspace/src/main/webapp/resources/extra/Module_1.css [hunk 1]
```diff
@@ -188,6 +188,16 @@ a, a:hover, a:focus {
...
+<<<<<<< HEAD
+@media only screen and (max-width: 800px) {
+    /* For mobile phones: */
+    [class*="exit_to_space_Class"] {
+        top: 45%;
+    }
+}
+
+=======
+>>>>>>> develop
```
Comment (jdamerow) [inline vspace/src/main/webapp/resources/extra/Module_1.css:200]: ?

**Review by jdamerow — CHANGES_REQUESTED** [review]

File: vspace/src/main/java/edu/asu/diging/vspace/web/staff/DashboardController.java [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
+    @RequestMapping("/staff/dashboard")
+    public String displayDashboard(Model model) {
+        List<Space> recentSpaces = spaceRepo.findTop5ByOrderByCreationDateDesc();
+        List<Module> recentModules = moduleRepo.findTop5ByOrderByCreationDateDesc();
+        model.addAttribute("recentSpaces", recentSpaces);
+        model.addAttribute("recentModules", recentModules);
+        return "staff/dashboard/dashboard";
+    }
+
+}
```
[the remark this answers is not in the document]
Comment (jdamerow) [reply vspace/src/main/java/edu/asu/diging/vspace/web/staff/DashboardController.java:32]: ?

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
- navigation items in left navbar need to be centered
- logout icon still has a grey background
- link that has hover title "Exhibition Space" should say "Go to exhibition"
- contents of pages doesn't seem to be centered (there is more space on the left from navbar to content than on the right)
- bottom of side navbar, where it says "Virtual Spaces 2.0  DigInG 2020" is breaking after the first 20:
```
Virtual Spaces 2.0  DigInG 20
20
```

**Comment by jdamerow** [comment]
This ticket is just for the layout tweaks, right? It shouldn't have any text block files.

**Comment by AvirupBiswas** [comment]
> This ticket is just for the layout tweaks, right? It shouldn't have any text block files.

Yes the ticket was just for layout tweaks but after resolving the merge conflict I got these changes.

**Pull request closed without merging**